extensions/amp-analytics/0.1/vendors/taboola.json

```diff
@@ -0,0 +1,20 @@
+{
+  "vars": {
+    "aid": ""
+  },
+  "requests": {
+    "url": "https://3p.ampproject.net/custom/amp-analytics-taboola.html?url=${canonicalUrl}&aid=${aid}"
+  },
+  "triggers": {
+    "pageview": {
+      "on": "visible",
+      "request": "url",
+      "iframePing": true
```

Comment:
`iframePing` is used as a workaround for vendors transitioning to amp-analytics integration. Is this true for your case?

Reply:
@micajuine-ho did not understand that. My bad. So removed that configuration.